script2.js

```diff
@@ -0,0 +1,94 @@
+const API = 'https://pokeapi.co/api/v2/pokemon/'
+const wrapper = document.querySelector('wrapper')
+const main = document.querySelector('main')
+const pokemonOne = document.querySelector('.pokemonOne')
+const backCards = Array.from(document.querySelectorAll('.backCard'))
+const next = document.querySelector('.next')
+const previous = document.querySelector('.previous')
+const page = document.querySelector('.currentPage');
+let localDataBase = ['empty'];
+let pokemonsFetchesSoFar = 0;
+let currentPage = 0;
+let c = pokemonsFetchesSoFar; // counter
+let allPages = [];
+
+async function fetchPokemons() {
+    let pokemonsFetched = [];
+
+    for (let i = 0; i < 3; i++) {
```

Comment:
Rozbić na więcej lini oraz dodać obsługę błędów
```suggestion
        try {
            const response = await fetch(API + c)
            const pokemonData = await response.json()
            pokemonsFetched.push(pokemonData);
        } catch (error) {
            alert(error.message)
            throw new Error(error.message)
        }
```